install/kubernetes/cilium/templates/hubble/servicemonitor.yaml

```
@@ -38,4 +38,7 @@ spec:
     metricRelabelings:
     {{- toYaml . | nindent 4 }}
     {{- end }}
+{{- if .Values.hubble.metrics.serviceMonitor.jobLabel }}
+  jobLabel: {{ .Values.prometheus.serviceMonitor.jobLabel | quote }}
```

did you meant
```suggestion
  jobLabel: {{ .Values.hubble.metrics.serviceMonitor.jobLabel | quote }}
```
instead?

---

I did and you should see it fixed and correct